spec/support/factory_girl.rb

```diff
@@ -13,6 +13,9 @@
     state 'ma'
     zip '12345'
     phone '1234567890'
+    profile_photo { Rack::Test::UploadedFile.new(
```

Comment:
Avoid using `{...}` for multi-line blocks.<br>Block body expression is on the same line as the block start.